homeassistant/components/automation/__init__.py

```diff
@@ -585,8 +589,12 @@ def log_cb(level, msg, **kwargs):
         variables = None
         if self._trigger_variables:
             try:
+                states = self.hass.states.get(self.entity_id)
+                this = states.as_dict() if states else {"entity_id": self.entity_id}
                 variables = self._trigger_variables.async_render(
-                    self.hass, None, limited=True
+                    self.hass,
+                    {"this": this},
```

Comment:
It's confusing to add the entire state to the trigger variables because the data will be old when the automation triggers.
Maybe just add the `entity_id` here?

Reply:
On second thought, never mind